src/components/SelectBox/Select/Trigger.tsx

```diff
@@ -0,0 +1,130 @@
+import type { ColorScheme, Size, StyledSelectProps } from '../index'
+import { SelectContext, SelectStyleContext } from './index'
+import { hexToCSSFilter } from 'hex-to-css-filter'
+import { ICON } from '@constants'
+import { Image } from '@components'
+import type { ReactElement } from 'react'
+import styled from '@emotion/styled'
+import type { Theme } from '@emotion/react'
+import { useContext } from 'react'
+
+type GetFontColorParams = Omit<StyledSelectProps, 'isSelected'> & {
+  theme: Theme
+}
+type GetFontColor = (params: GetFontColorParams) => string
+type ApplyIconColor = (colorScheme: ColorScheme, theme: Theme) => string
+type ApplySize = (size: Size, theme: Theme) => string
+type ApplyColorScheme = (colorScheme: ColorScheme, theme: Theme) => string
+
+export const Trigger = (): ReactElement => {
+  const { placeholder, setIsOpen, value, options } = useContext(SelectContext)
+  const { size, colorScheme } = useContext(SelectStyleContext)
+  const text = options.find(option => option.value === value)?.text || ''
```

Comment:
값이 없을 때 placeholder를 바로 넣어주는건 어떨까용 ?

```suggestion
  const text = options.find(option => option.value === value)?.text || placeholder
```

Reply:
selectBox 리팩터링은 요깃!! [08b16440](https://github.com/price-offer/offer-design-system/commit/08b16440e41486cc9ef958ed73d82e19eb810c84)